models/task.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Task extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      this.belongsTo(models.User, {foreignKey: 'userId'})
+    }
+  };
+  Task.init({
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notNull: {
```

Comment:
apakah null sama dengan string kosong? kalau user masukin string kosong masih bisa ga? handle boy

Reply:
baik kak, kurang not empty